packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -151,122 +151,126 @@ const App = (props) => {
+                <IntlProvider
+                    onError={(err) => {
+                        if (err.code === 'MISSING_TRANSLATION') {
+                            // NOTE: Remove the console error for missing translations during development,
+                            // as we knew translations would be added later.
+                            console.warn('Missing translation', err.message)
+                            return
+                        }
+                        throw err
+                    }}
+                    locale={targetLocale}
+                    messages={messages}
+                    // For react-intl, the _default locale_ refers to the locale that the inline `defaultMessage`s are written for.
+                    // NOTE: if you update this value, please also update the following npm scripts in `template-retail-react-app/package.json`:
+                    // - "extract-default-translations"
+                    // - "compile-translations:pseudo"
+                    defaultLocale={DEFAULT_LOCALE}
+                >
+                    <CategoriesProvider treeRoot={allCategories} locale={targetLocale}>
+                        <CurrencyProvider currency={currency}>
+                            <Seo>
+                                <meta name="theme-color" content={THEME_COLOR} />
+                                <meta
+                                    name="apple-mobile-web-app-title"
+                                    content={DEFAULT_SITE_TITLE}
+                                />
+                                <link
+                                    rel="apple-touch-icon"
+                                    href={getAssetUrl('static/img/global/apple-touch-icon.png')}
+                                />
+                                <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
+
+                                {/* Urls for all localized versions of this page (including current page)
+                                For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
+                                {site.l10n?.supportedLocales.map((locale) => (
+                                    <link
+                                        rel="alternate"
+                                        hrefLang={locale.id.toLowerCase()}
+                                        href={`${appOrigin}${buildUrl(location.pathname)}`}
+                                        key={locale.id}
+                                    />
+                                ))}
+                                {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
                                 <link
                                     rel="alternate"
-                                    hrefLang={locale.id.toLowerCase()}
+                                    hrefLang={site.l10n.defaultLocale.slice(0, 2)}
```

Comment:
can `site.l10n.defaultLocale` ever be uppercase? I thought that was configurable in BM and could maybe be lowercase if a user configures it that way?

Reply:
not relevant to this PR, the diff is messed up because of the wrapping